fastchat/serve/openai_api_server.py

```diff
@@ -19,7 +20,7 @@
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import StreamingResponse, JSONResponse
 from fastapi.security.http import HTTPAuthorizationCredentials, HTTPBearer
-import httpx
+import aiohttp
```

Comment:
1. Sort the packages alphabetically
2. Declare this dependency in `pyproject.toml`

Reply:
> 1. Sort the packages alphabetically
> 2. Declare this dependency in `pyproject.toml`

Done